source/d3d11/draw_call_tracker.cpp

```diff
@@ -1,105 +1,168 @@
 #include "draw_call_tracker.hpp"
 #include "runtime.hpp"
 #include <math.h>
+#include "log.hpp"
 
 namespace reshade::d3d11
 {
 	void draw_call_tracker::merge(const draw_call_tracker& source)
 	{
-		_counters.vertices += source.vertices();
-		_counters.drawcalls += source.drawcalls();
+		_global_counter.vertices += source.total_vertices();
+		_global_counter.drawcalls += source.total_drawcalls();
 
-		for (auto source_entry : source._counters_per_used_depthstencil)
+		for (auto &[depthstencil, snapshot] : source._counters_per_used_depthstencil)
 		{
-			const auto destination_entry = _counters_per_used_depthstencil.find(source_entry.first);
+			const auto destination_entry = _counters_per_used_depthstencil.find(depthstencil);
```

Comment:
This isn't used? Looks like the original intention was to write `destination_entry->second.counter.vertices += ...` below instead of `_counters_per_used_depthstencil[depthstencil].counter.vertices += ...`, but that's not the case here. Same thing a couple of lines below.